packages/fxa-event-broker/package.json

```diff
@@ -8,10 +8,10 @@
     "audit": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
     "lint:tslint": "./node_modules/tslint/bin/tslint -p .",
     "watch": "tsc -w",
-    "test": "FIRESTORE_EMULATOR_HOST=localhost:8006 FIRESTORE_PROJECT_ID=fx-event-broker ./node_modules/mocha/bin/mocha -r ts-node/register test/**/*.spec.ts test/**/**/*.spec.ts",
+    "test": "mocha -r ts-node/register test/**/*.spec.ts test/**/**/*.spec.ts",
     "start-prod": "npm run build && node ./dist/bin/worker.js",
-    "start": "npm run build && NODE_ENV=development PUBSUB_EMULATOR_HOST=localhost:8005 FIRESTORE_EMULATOR_HOST=localhost:8006 node ./dist/bin/workerDev.js",
-    "start-dev-debug": "NODE_ENV=development PUBSUB_EMULATOR_HOST=localhost:8005 FIRESTORE_EMULATOR_HOST=localhost:8006 nodemon --legacy-watch --watch ./dist --inspect=0.0.0.0:9180 --nolazy ./dist/bin/workerDev.js"
+    "start": "npm run build && NODE_ENV=development node ./dist/bin/workerDev.js",
+    "start-dev-debug": "NODE_ENV=development nodemon --legacy-watch --watch ./dist --inspect=0.0.0.0:9180 --nolazy ./dist/bin/workerDev.js"
```

Comment:
I was under the impression those env vars need to be set or it won't use the emulators. That isn't the case after all?

Reply:
It looks like the ports are hard coded in the tests. I didn't want to dig too deep.

Reply:
i put the env vars back